core/tests/snapshots.rs

```diff
@@ -788,7 +788,7 @@ mod tests {
             .find(|elem| elem.slot == slot)
             .ok_or_else(|| Error::new(ErrorKind::Other, "did not find snapshot with this path"))?;
         let storages = {
-            let mut storages = bank.get_snapshot_storages();
+            let mut storages = bank.get_snapshot_storages(Some(incremental_snapshot_base_slot));
             snapshot_utils::filter_snapshot_storages_for_incremental_snapshot(
```

Comment:
The call to `filter_snapshot_storages_for_incremental_snapshot()` will be removed in the next PR. I am trying to keep PRs as minimal as possible :)